src/components/login.tsx

```diff
@@ -22,27 +21,13 @@ const Login: FunctionComponent<LoginProps> = () => {
   }
 
   async function submitPassword() {
-    const userCredentials = {
-      username,
-      password,
-    };
-
-    const result = await fetch("https://tva-backend.herokuapp.com/login", {
-      method: "POST",
-      body: JSON.stringify(userCredentials),
-      headers: {
-        "Content-Type": "application/json",
-      },
-      credentials: "include",
-    });
-    const data = await result.json();
-
-    if (data.success) {
-      setAuth({ user: userCredentials.username, userId: data.userId });
-      navigate("/", { replace: true });
-    } else {
-      setLoginFailed(true);
-    }
+    signin(username, password)
```

Comment:
We call the `signin` here, if there is an error we can catch it and change the state.